src/main/kotlin/AddCalculator.kt

```diff
@@ -0,0 +1,26 @@
+class AddCalculator {
+    private val inputValidator = InputValidator()
+    private val inputParser = InputParser()
+
+    fun sum(input: String?): Int {
+        if (input.isNullOrBlank()) {
+            return 0
+        }
+
+        val tokens = inputParser.parse(input)
+        checkValidNumbers(tokens)
+
+        val numbers = tokens.map { it.toInt() }
+        return sum(numbers)
```

Comment:
sum()함수를 굳이 호출할 필요가있을까요?

```suggestion
        return numbers.sum()
```